data/SavvasStephanides.json

```diff
@@ -0,0 +1,28 @@
+{
+  "name": "Savvas Stephanides",
+  "displayStatsPublic": true,
+  "type": "personal",
+  "bio": "Developer circa 2001",
+  "links": [
+    {
+      "name": "Twitter",
+      "url": "https://twitter.com/SavvasStephnds",
+      "icon": "FaTwitter"
+    },
+    {
+      "name": "Instagram",
+      "url": "https://www.instagram.com/savvascodes/",
+      "icon": "FaInstagram"
+    },
+    {
+      "name": "Github",
+      "url": "https://github.com/SavvasStephanides",
+      "icon": "FaGithub"
+    },
+    {
+      "name": "LinkedIn",
+      "url": "https://www.linkedin.com/in/savvasstephanides",
+      "icon": "FaGithub"
```

Comment:
Change the icon to `FaLinkedin`

Reply:
Thank you Priyankar! Didn't notice that! 🙏